builtin/logical/pki/backend_test.go

```diff
@@ -4034,7 +4034,7 @@ func getParsedCrl(t *testing.T, client *api.Client, mountPoint string) *pkix.Cer
 }
 
 func getParsedCrlForIssuer(t *testing.T, client *api.Client, mountPoint string, issuer string) *pkix.CertificateList {
-	path := fmt.Sprintf("/v1/%v/issuer/%v/crl", mountPoint, issuer)
+	path := fmt.Sprintf("/v1/%v/issuer/%v/crl/der", mountPoint, issuer)
```

Comment:
Curious why did this path change? If we needed to add a der to it that would mean we aren't backwards compat anymore?